src/ethereum_spec_tools/evm_tools/t8n/__init__.py

```diff
@@ -357,7 +357,9 @@ def apply_body(self) -> None:
                     env, tx
                 )
 
-                if self.fork.is_after_fork("ethereum.cancun"):
+                if self.fork.is_after_fork(
+                    "ethereum.cancun"
+                ) and not self.fork.is_after_fork("ethereum.prague"):
                     blob_gas_used += self.fork.calculate_total_blob_gas(tx)
```

Comment:
Incrementing the `blob_gas_used` has to happen post Prague as well I believe. It's only the `MAX_BLOB_GAS_PER_BLOCK` check that is removed

Reply:
Fixed